src/main/java/htsjdk/samtools/util/SamConstants.java

```diff
@@ -0,0 +1,11 @@
+package htsjdk.samtools.util;
+
+/**
+ * A set of constants defined in the sam-spec (https://github.com/samtools/hts-specs) that need
+ * to be referenced in code.
+ */
+public class SamConstants {
```

Comment:
make this final...
```suggestion
 public final class SamConstants {
```

and I'd give it a private constructor as well, but I know that's more controversial...